source/extensions/common/wasm/BUILD

```diff
@@ -59,21 +59,9 @@ envoy_cc_library(
 
 envoy_cc_library(
     name = "wasm_interoperation_lib",
```

Comment:
I think we no longer need this lib

Reply:
Thanks. Removed this rule and fixed the comment in `cel_state.cc`.